zokrates_core_test/tests/tests/compare_min_to_max.zok

```diff
@@ -1,10 +0,0 @@
-from "field" import FIELD_MAX;
-
-// /!\ should be called with a = 0
-// as `|a - FIELD_MAX| < 2**(N-2)` the comparison should succeed
-
-def main(field a) -> bool {
-    field p = FIELD_MAX + a;
-    // we added a = 0 to prevent the condition to be evaluated at compile time
-    return a < p;
```

Comment:
The other tests down below all do `assert(condition)`. Do we cover all cases which just use the condition without asserting it, for example returning it? They are handled differently in code generation so they should be also tested separately.